roxctl/common/printer/tabularprinter_factory.go

```diff
@@ -1,14 +1,34 @@
 package printer
 
 import (
+	"slices"
 	"strings"
 
+	"github.com/cloudflare/cfssl/log"
```

Comment:
Not sure if this was intended. Consider using `cmd.env.Logger()`.

Reply:
Switched to using `cmd.PrintErrf()` if that's alright(?) `cmd.Env.Logger()` is only available for different type of commands we use in roxctl, not cobra.